cluster-operator/src/main/java/io/strimzi/operator/cluster/Main.java

```diff
@@ -167,16 +167,13 @@ static CompositeFuture run(Vertx vertx, KubernetesClient client, PlatformFeature
             List<Future> futures = new ArrayList<>();
             ClusterRoleOperator cro = new ClusterRoleOperator(vertx, client);
 
-            Map<String, String> clusterRoles = new HashMap<String, String>() {
-                {
-                    put("strimzi-cluster-operator-namespaced", "020-ClusterRole-strimzi-cluster-operator-role.yaml");
-                    put("strimzi-cluster-operator-global", "021-ClusterRole-strimzi-cluster-operator-role.yaml");
-                    put("strimzi-kafka-broker", "030-ClusterRole-strimzi-kafka-broker.yaml");
-                    put("strimzi-entity-operator", "031-ClusterRole-strimzi-entity-operator.yaml");
-                    put("strimzi-topic-operator", "032-ClusterRole-strimzi-topic-operator.yaml");
-                    put("strimzi-kafka-client", "033-ClusterRole-strimzi-kafka-client.yaml");
-                }
-            };
+            Map<String, String> clusterRoles = new HashMap<>();
```

Comment:
Maybe you can set the map size here since it looks like we now the size upfront?

Reply:
Please help me review again